bokeh/charts/builder/bar_builder.py

```diff
@@ -36,12 +36,23 @@
 
 
 def Bar(values, cat=None, stacked=False, xscale="categorical", yscale="linear",
-        xgrid=False, ygrid=True, **kw):
+        xgrid=False, ygrid=True, continuous_range=None, **kw):
+
+    if continuous_range and not isinstance(continuous_range, Range1d):
```

Comment:
Are you sure?

`y_range` & `x_range` are only checked to be an instance of Range.

Checking `continuous_range` for Range1d (and, in the future, `categorical_range` for an instance of FactorRange) seems like a useful check to provide feedback to user.